package.json

```diff
@@ -7,7 +7,8 @@
     "node": ">=5.0.0"
   },
   "scripts": {
-    "postinstall": "bower install; gulp",
+    "postinstall": "bower install; npm run generate-stackblitz; gulp",
+    "generate-stackblitz": "cp ./node_modules/@stackblitz/sdk/bundles/sdk.umd.js ./app/js/sdk.umd.js; cp ./node_modules/@stackblitz/sdk/bundles/sdk.umd.js.map ./app/js/sdk.umd.js.map;",
```

Comment:
Can we use a hosted version of the SDK instead? Not a huge fan of this one-off, manual copying from node_modules/.

```html
<script src="https://unpkg.com/@stackblitz/sdk/bundles/sdk.umd.js"></script>
```

Reply:
Sure, I'll just need to add this to the service worker

Reply:
spoke offline, decided it's best not to add SW